lib/Core.lua

```diff
@@ -25,6 +25,9 @@ Core.None = Symbol.named("None")
 -- Marker used to specify that the table it is present within is a component.
 Core.Element = Symbol.named("Element")
 
+-- The default "stack traceback" if element tracing is not enabled.
+Core._defaultSource = "\n\t<Use Roact.setGlobalConfig with the 'elementTracing' key to enable detailed tracebacks>\n"
```

Comment:
This could probably stand to have a better name, especially now that it's left the scope of the reconciler! Maybe something like `_elementTracebacksDisabledMessage`?

I'm starting to feel that we should collect some of the error messages together into a module in a future PR. 🤔